x-pack/plugins/rollup/public/visualize/agg_type_filter.js

```diff
@@ -17,6 +17,6 @@ aggTypeFilters.addFilter(
     }
     const aggName = aggType.name;
     const aggs = indexPattern.typeMeta.aggs;
-    return Object.keys(aggs).includes(aggName);
+    return aggName === 'count' || Object.keys(aggs).includes(aggName);
```

Comment:
Does this merit a comment since it looks like a special case?

Reply:
good catch, will add one